pkg/capabilities/consensus/ocr3/capability_test.go

```diff
@@ -136,7 +175,12 @@ func TestOCR3Capability_Registration(t *testing.T) {
 	cp := newCapability(s, fc, 1*time.Second, mockEncoderFactory, lggr)
 	require.NoError(t, cp.Start(ctx))
 
-	config, err := values.NewMap(map[string]any{"aggregation_method": "data_feeds_2_0"})
+	config, err := values.NewMap(map[string]any{
+		"aggregation_method": "data_feeds_2_0",
+		"aggregation_config": map[string]any{},
```

Comment:
@HenryNguyen5 This is what caused me to add a values.EmptyMap -- is this guaranteed to be present because of the schema validation? (I assume so)

Reply:
Yeah, its required to be specfied now, a YAML representation would be:
```yaml
aggregation_method: data_feeds_2_0
aggregation_config: {}
```